manifests/03-clusterrole.yaml

```diff
@@ -195,3 +195,4 @@ subjects:
 - kind: ServiceAccount
   name: operator
   namespace: openshift-insights
+  
```

Comment:
Nitpick: this whitespace change looks like unnecessary

Reply:
probably the wrong text editor configuration